charts/topolvm/values.yaml

```diff
@@ -401,10 +401,10 @@ controller:
       requiredDuringSchedulingIgnoredDuringExecution:
         - labelSelector:
             matchExpressions:
-              - key: app.kubernetes.io/name
+              - key: app.kubernetes.io/component
                 operator: In
                 values:
-                  - topolvm-controller
+                  - controller
```

Comment:
Since `app.kubernetes.io/component=controller` label is also used in cert-manager deployment, it is necessary to set other appropriate labels too.

Reply:
https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/
This document says `app.kubernetes.io/part-of` is to refer to a higher-level application name. But this component is just one of the topolvm application, so the label is not suitable, IMO.
I think it is better to use `app.kubernetes.io/name` instead. Do you have a reason to use the `part-of` label?

Reply:
memo: the value for `app.kubernetes.io/name` is configurable by values.yaml, so we need to use the template variable. To do that, the `affinity` section type is now string.